frontend/public/components/resource-quota.jsx

```diff
@@ -253,22 +322,50 @@ export const hasComputeResources = (resourceTypes) => {
   return _.intersection(resourceTypes, chartResourceTypes).length > 0;
 };
 
-const Details = ({ obj: rq }) => {
+const Details = ({ obj: rq, match }) => {
   const { t } = useTranslation();
   const resourceTypes = getQuotaResourceTypes(rq);
   const showChartRow = hasComputeResources(resourceTypes);
   const scopes = _.get(rq, ['spec', 'scopes']);
-  const text = isClusterQuota(rq)
-    ? t('public~ClusterResourceQuota details')
-    : t('public~ResourceQuota details');
+  const kind = quotaKind(rq);
```

Comment:
This function name and variable name are misleading. This is not a kind, it's a reference, which is something different.